src/amchart/CommonSerial.js

```diff
@@ -101,7 +101,7 @@
     CommonSerial.prototype.publish("yAxisType", "linear", "set", "Y-Axis Type", ["none", "linear", "pow", "log", "time"],{tags:["Intermediate","Shared"]});
     CommonSerial.prototype.publish("xAxisType", "ordinal", "set", "X-Axis Type", ["ordinal", "linear", "time"]);
 
-    CommonSerial.prototype.publish("yAxisTickFormat", "s", "string", "Y-Axis Tick Format");
+    CommonSerial.prototype.publish("yAxisTickFormat", ".2f", "string", "Y-Axis Tick Format");
```

Comment:
@GordonSmith 
it's not, i dont think the Axis tick of S is the same as the others either. (however if we dont do any rounding the chart cursor when enabled is to wide and gets clipped for intermediate values) ... (what should we institute as the defaults for the tickformat)?